### PR TITLE
Update ABIs

### PR DESCRIPTION
**Pull request by troggy**
- Update bridge/operator/exitHandler ABIs to the relevant versions
- Get rid of hardcoded governable ABIs, but [concat](https://github.com/leapdao/bridge-ui/pull/75/files#diff-ac7fc99103d4b708ab09b4002085d2c2R6) all the hub contracts ABIs instead

### CODE REVIEW

File: src/utils/abis/index.ts [hunk 1]
```diff
@@ -1,13 +1,16 @@
...
-export { default as governance } from './governance';
-
-import { default as newBridge } from './newBridgeAbis';
+import { default as bridge } from './bridge';
+import { default as operator } from './operator';
+import { default as exitHandler } from './exitHandler';
 import { default as proxy } from './proxy';
 
-const governable = newBridge.concat(proxy);
+const governable = (proxy as any)
```
Comment (sunify) [inline src/utils/abis/index.ts]: import { ABIDefinition } from 'web3/eth/abi';

File: src/utils/abis/proxy.ts [hunk 2]
```diff
@@ -55,4 +57,4 @@ export default [
     stateMutability: 'nonpayable',
     type: 'function',
   },
-];
+] as unknown as ABIDefinition[];
```
Comment (troggy) [inline src/utils/abis/proxy.ts]: 🤷‍♂️

**Comment by troggy** [comment]
@sunify is it any good now?

**Comment by sunify** [comment]
Yes

**Pull request merged**